examples/send_image.py

```diff
@@ -1,16 +1,23 @@
-from atproto import Client
+from atproto import Client, models
 
 
 def main() -> None:
     client = Client()
-    client.login('my-handle', 'my-password')
+    client.login("my-handle", "my-password")
 
     # replace the path to your image file
-    with open('cat.jpg', 'rb') as f:
+    with open("cat.jpg", "rb") as f:
         img_data = f.read()
 
-    client.send_image(text='Post with image from Python', image=img_data, image_alt='Text version of the image (ALT)')
+    aspect_ratio = models.AppBskyEmbedDefs.AspectRatio(height=100, width=100)
```

Comment:
let's cover this line with some comments

Reply:
Added comments.